### PR TITLE
Updated What's New on January 27, 2020

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Updated the [What's New](https://devdocs.magento.com/whats-new.html) with changes published since November 28, 2019.

## Affected DevDocs pages

- https://devdocs.magento.com/whats-new.html

### CODE REVIEW

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,148 @@ description: |
...
+updated: Mon Jan 27 21:17:38 2020
 entries:
+- description: Added `@method` and `@link` information to [Coding Standards](https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html).
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: January 17, 2020
+  link: https://github.com/magento/devdocs/pull/6223
+- description: Updated the [Pro architecture topic](https://devdocs.magento.com/cloud/architecture/pro-architecture.html#pro-cluster-scaling)
+    topic in the _Cloud Guide_ to include information about scaling the architecture.
+  versions: ''
```
Comment (dshevtsov) [inline src/_data/whats-new.yml]: missing versions

**Comment by dobooth** [comment]
Is this supposed to be merged into small-changes? I don't see in the history where that base branch was changed from master.

**Comment by jeff-matthews** [comment]
running tests

File: src/_data/whats-new.yml (hunk 1 above)
Comment (dshevtsov) [reply src/_data/whats-new.yml]: it should be 2.x then I guess

Comment (dshevtsov) [reply src/_data/whats-new.yml]: or it could use the cloud release line versioning, as we do for bundled extensions

**Pull request merged**